R/model-design-tools.R

```diff
@@ -265,23 +265,27 @@ enw_one_hot_encode_feature <- function(metaobs, feature, contrasts = FALSE) {
 #' @family modeldesign
 #' @export
 #' @importFrom purrr map
+#' @importFrom rlang abort
 #' @examples
 #' metaobs <- data.frame(week = 1:2)
 #' enw_add_cumulative_membership(metaobs, "week")
 #'
-#' metaobs <- data.frame(week = 1:3, .group = c(1,1,2))
+#' metaobs <- data.frame(week = 1:3, .group = c(1, 1, 2))
```

Comment:
@seabbs the commas example - i would expect linter to complain about this.

Reply:
yes me as well but it doesn't appear to

Reply:
this seems to be the result of an automatic action of my VSCode linter (I'm normally using Rstudio, but it's currently unstable so switched to VSCode). Are you happy to keep these in? Or should I try and remove them again?